platform/ios/BUILD.bazel

```diff
@@ -441,6 +441,32 @@ objc_library(
     ],
 )
 
+objc_library(
+    name = "benchapp",
+    copts = MAPLIBRE_FLAGS,
+    srcs = [
+        "platform/ios/benchmark/main.m",
+        "platform/ios/benchmark/MBXBenchAppDelegate.m",
+        "platform/ios/benchmark/MBXBenchViewController.mm",
+        "platform/ios/benchmark/locations.cpp",
+    ],
+    hdrs = [
+        "platform/ios/benchmark/MBXBenchAppDelegate.h",
+        "platform/ios/benchmark/MBXBenchViewController.h",
+        "platform/ios/benchmark/locations.hpp",
+    ],
+    includes = [
+        "platform/darwin/app",
+        "platform/darwin/src",
+        "platform/ios/src",
+        "platform/ios/src",
+        "../../include",
```

Comment:
it's needed because [MLNMapView_Private.h](https://github.com/maplibre/maplibre-native/blob/topic/drawable-fixes/platform/ios/platform/ios/src/MLNMapView_Private.h) does 
```
#include <mbgl/util/size.hpp>
```

Reply:
Can't you add `mbgl-core` to `deps`?

Reply:
yes, this works! Thanks!